### ISSUE TITLE
'Pool' object has no attribute 'default'

### ISSUE THREAD

**Issue by m1nde** [sdispater/poetry#1163]
- [x] I am on the [latest](https://github.com/sdispater/poetry/releases/latest) Poetry version: 1.0.0a4
- [x] I have searched the [issues](https://github.com/sdispater/poetry/issues) of this repo and believe that this is not a duplicate.
- [x] If an exception occurs when executing a command, I executed it again in debug mode (`-vvv` option).

- **OS version and name**: macOS 10.14
- **Poetry version**: 1.0.0a4

Using poetry 1.0.0a3 and 1.0.0a4, adding a package from a private repository gives the following error:
```
  - Installing my-private-package (0.0.1)

[AttributeError]
'Pool' object has no attribute 'default'
````

Full stack trace:
```
Traceback (most recent call last):
  File "/Users/0x72/.poetry/lib/poetry/_vendor/py3.7/clikit/console_application.py", line 131, in run
    status_code = command.handle(parsed_args, io)
  File "/Users/0x72/.poetry/lib/poetry/_vendor/py3.7/clikit/api/command/command.py", line 112, in handle
    status_code = self._do_handle(args, io)
  File "/Users/0x72/.poetry/lib/poetry/_vendor/py3.7/clikit/api/command/command.py", line 160, in _do_handle
    return getattr(handler, handler_method)(args, io, self)
  File "/Users/0x72/.poetry/lib/poetry/_vendor/py3.7/cleo/commands/command.py", line 92, in wrap_handle
    return self.handle()
  File "/Users/0x72/.poetry/lib/poetry/console/commands/add.py", line 158, in handle
    status = installer.run()
  File "/Users/0x72/.poetry/lib/poetry/installation/installer.py", line 75, in run
    self._do_install(local_repo)
  File "/Users/0x72/.poetry/lib/poetry/installation/installer.py", line 292, in _do_install
    self._execute(op)
  File "/Users/0x72/.poetry/lib/poetry/installation/installer.py", line 308, in _execute
    getattr(self, '_execute_{}'.format(method))(operation)
  File "/Users/0x72/.poetry/lib/poetry/installation/installer.py", line 333, in _execute_install
    self._installer.install(operation.package)
  File "/Users/0x72/.poetry/lib/poetry/installation/pip_installer.py", line 58, in install
    if repository.name != self._pool.default.name:
```

This does not happen with packages from the official PyPI repo. It happens with every package from our private JFrog server. It does not happen with poetry version 1.0.0a2.

**Comment by areeh** [comment]
Same issue when updating from 1.0.0a2 to 1.0.0a3

**Comment by droserasprout** [comment]
`self._pool.default` is just indicator showing if default repository exists, not a link to default repository itself like expected in this code.

**Comment by pbzweihander** [comment]
I think commit https://github.com/sdispater/poetry/commit/6f4aa21ce65dd719cfc98951d207ff39cd1d94cf fixed this.

**Comment by sdispater** [comment]
This should be fixed in the `1.0.0a5` release.
